spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -233,3 +233,35 @@ include::{snippets}/task-executions-documentation/launch-task/curl-request.adoc[
 ===== Response structure
 
 include::{snippets}/task-executions-documentation/launch-task/http-response.adoc[]
+
+[[resources-metrics-stream-applications]]
+=== Metrics for Stream Applications
+
+This Rest endpoint exposes metrics for stream applications. This Rest enpoint requires
+the https://github.com/spring-cloud/spring-cloud-dataflow-metrics-collector[Metrics Collector]
+application to be running as a separate service. If not running, this endpoint will
+return an empty response.
+
+NOTE: In order to learn more about the Metrics Collector, please also refer to chapter
+<<configuration-monitoring-deployed-applications>>
+
+For example, a typical request may look like:
+
+include::{snippets}/metrics-for-stream-apps-without-collector-documentation/get-metrics-without-collector-running/curl-request.adoc[]
+
+This Rest endpoint uses https://github.com/Netflix/Hystrix[Hystrix] via the
+https://github.com/spring-cloud/spring-cloud-netflix[Spring Cloud Netflix] project
+under the covers to make a proxy Http request to the
+Metrics Collector. Therefore, the endpoint will not generate an error in case the
+Metrics Collector is not running, but rather gracefully degrade and return an
+empty response such as the following:
+
+include::{snippets}/metrics-for-stream-apps-without-collector-documentation/get-metrics-without-collector-running/http-response.adoc[]
+
+However, if metrics are being collected and the Metrics Collector is running you
+should see a response similar to the following:
+
+include::{snippets}/metrics-for-stream-apps-documentation/get-metrics-with-collector-running/http-response.adoc[]
+
+The metrics data returned in the example is based on the example stream definition created
```

Comment:
moved to before the REST output and mention two instances of each app were deployed.